src/SDKs/_metadata/cognitiveservices_data-plane_ComputerVision.txt

```diff
@@ -1,18 +1,35 @@
 ﻿Installing AutoRest version: latest
+
+> autorest@2.0.4283 preinstall C:\Users\cthrash.REDMOND\AppData\Roaming\npm\node_modules\autorest
+> node ./preinstall-check
+
+C:\Users\cthrash.REDMOND\AppData\Roaming\npm\autorest -> C:\Users\cthrash.REDMOND\AppData\Roaming\npm\node_modules\autorest\dist\app.js
++ autorest@2.0.4283
+updated 1 package in 3.21s
+
 AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/ComputerVision/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=E:\azure-sdk-for-net\src\SDKs
-2018-07-23 17:11:16 UTC
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/ComputerVision/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=F:\src\github\cthrash\azure-sdk-for-net\src\SDKs
+AutoRest code generation utility [version: 2.0.4283; node: v8.9.3]
+(C) 2018 Microsoft Corporation.
+https://aka.ms/autorest
+C:\Users\cthrash.REDMOND\.autorest\@microsoft.azure_autorest-core@2.0.4283\autorest-core -> C:\Users\cthrash.REDMOND\.autorest\@microsoft.azure_autorest-core@2.0.4283\node_modules\@microsoft.azure\autorest-core\dist\app.js
+C:\Users\cthrash.REDMOND\.autorest\@microsoft.azure_autorest-core@2.0.4283\autorest-language-service -> C:\Users\cthrash.REDMOND\.autorest\@microsoft.azure_autorest-core@2.0.4283\node_modules\@microsoft.azure\autorest-core\dist\language-service\language-service.js
+   Loading AutoRest core      'C:\Users\cthrash.REDMOND\.autorest\@microsoft.azure_autorest-core@2.0.4283\node_modules\@microsoft.azure\autorest-core\dist' (2.0.4283)
+   Installing AutoRest extension '@microsoft.azure/autorest.csharp' (~2.3.79)
+   Installing AutoRest extension '@microsoft.azure/autorest.modeler' (2.3.55)
+
+2018-07-30 21:43:07 UTC
 1) azure-rest-api-specs repository information
 GitHub fork: Azure
 Branch:      master
-Commit:      9754d04e95bc4ffe16186aa76f652773923f5e10
+Commit:      1a8570aeb1c216495ab782ec4e19db30ee2cacbf
 
 2) AutoRest information
 Requested version: latest
-Bootstrapper version:    autorest@2.0.4282
+Bootstrapper version:    C:\Users\cthrash.REDMOND\AppData\Roaming\npm  `-- autorest@2.0.4283
```

Comment:
Please pull latest changes from psSdkJson6, run msbuild build.proj and regenerate the code